static/app/types/integrations.tsx

```diff
@@ -379,7 +379,7 @@ export type PlatformExternalIssue = {
 export type IssueConfigField = Field & {
   name: string;
   choices?: Choices;
-  default?: string | number;
+  default?: string | number | Array<string | number>;
```

Comment:
What does it mean for a default to be an array? How would we pick the default from the array? If there are multiple, options that could 'theoretically' be default, we should be documenting how we select which one we use in the event that multiple are present.